portfolio/src/main/java/com/google/sps/data/EntityConvertible.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+/** Interface that converts an Object to an Entity in Datastore. */
```

Comment:
Nit: I think you need to add that the implementing class need to provide a creator as well?

Reply:
Not so sure what you mean:)

Reply:
Fixed; add the comment